docs/7-vector-search/10-quantization.mdx

```diff
@@ -0,0 +1,80 @@
+# 🦸‍♂️ Vector quantization
+
+Vector quantization is a technique to reduce the number of bits required to represent a vector. This can help vastly improve vector search performance.
+
+However, before you enable quantization, note the following:
+
+1. Note the current vector search index size. You may do so via the Atlas UI.
+2. In the `searchBooks` function, add a couple of lines of code to log the executionStats. Your `searchBooks` function should look like the following:
+
+<details>
+<summary>Click to expand</summary>
+<div>
+
+```javascript
+public async searchBooks(query: string): Promise<Book[]> {
+  const vector = await getEmbeddings(query);
+  const aggregationPipeline = [
+    {
+      $vectorSearch: {
+        queryVector:  vector,
+        path: 'embeddings',
+        numCandidates: 100,
+        index: 'vectorsearch',
+        limit: 10,
+      }
+    }
+  ];
+  //log executionStats
+  const result = await collections?.books?.aggregate(pipeline).explain();
```

Comment:
The parameter has to be `aggregationPipeline`.
Shouldn't we explicitly set `"executionStats"` for the explain call?

```suggestion
  const result = await collections?.books?.aggregate(aggregationPipeline).explain("executionStats");
```